lib/resty/jwt.lua

```diff
@@ -630,6 +648,10 @@ end
 
 
 function _M.verify(self, secret, jwt_str, leeway, encrypted)
+  if not is_nil_or_positive_number(leeway) then
+    error("'leeway' is expected to be a positive number of seconds.")
```

Comment:
I went this way because I was seeing this as a bug (ie. a developer mistake, a misconfiguration).

Unless I've overlooked something, everything that's returned through the `reason` is related to something wrong at the jwt level, not at the server level which runs the library.

Thoughts?